api/api.go

```diff
@@ -53,6 +53,8 @@ type Configer interface {
 // guarantees reliable and secure message delivery to the replica.
 // Message delivery delay is assumed not to grow indefinitely. This
 // assumption has to be satisfied to ensure the liveness of the system.
+// ReplicaMessageStreamHandler never fails except when incorrect replica
+// ID is passed.
```

Comment:
I believe, there should be a comma before 'except'. (Although, I'm totally not an expert in English, so I might be wrong.)